content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -113,11 +113,11 @@ kubectl create -f deploy/kubernetes/snapshot-controller
 
 2. Ensure that you have created a namespace where you want to install the driver. You can run `kubectl create namespace vxflexos` to create a new one.
 
-3. Check `helm/csi-vxflexos/driver-image.yaml` and confirm the driver image points to a new image.
+3. Check `helm/csi-vxflexos/values.yaml` and confirm the driver image points to a new image.
```

Comment:
might be a bit misleading now, since there is no driver image field. 
Maybe say something like:
"Ensure the image you want to install is in form <registry>/csi-vxflexos:v<version>"

Reply:
Ahh, good point. What do you think about moving this warning to the values file creation step as well as clarifying it like you suggested? It seems to me like it might make more sense there

Reply:
I agree, I also think that the "version" field in values table should have updated description, since it is also used to pull images as well as verify helm chart version

Reply:
@JacobGros ight addressed comments

Reply:
LGTM